project-management/src/main/java/life/qbic/projectmanagement/application/sample/SampleMetadata.java

```diff
@@ -49,12 +54,25 @@ public static SampleMetadata createUpdate(SampleId sampleId,
       OntologyTerm specimen,
       OntologyTerm analyte,
       String comment,
+      Map<ConfoundingVariableInformation, String> confoundingVariables,
       String experimentId) {
     return new SampleMetadata(sampleId, sampleCode, sampleName, analysisToBePerformed,
         biologicalReplicate, experimentalGroupId, species, specimen, analyte, comment,
+        confoundingVariables,
         experimentId);
   }
 
+  public static SampleMetadata fromNewToExisting(SampleId id, SampleMetadata sampleMetadata) {
```

Comment:
I am unsure about the naming here: the method just creates a new sample metadata record with the given sample ID. 
I suggest a naming that describes what it does, instead of assuming context. 

`addSampleID(SampleId id, SampleMetadata metadata)`